docs/release_process.md

```diff
@@ -12,52 +11,120 @@ Everyone is welcome to test the stabilization branch and propose fixes.
+
+    Hello all,
+
+    The release of Content version 0.1.65 is scheduled for December 2nd.
+    As part of the release process, a stabilization branch was created.
+    Issues and PRs that were not solved were moved to the 0.1.66 milestone.
+
+    Any bug fixes you would like to include in release 0.1.65 should be proposed
+    for the *stabilization-v0.1.65* and *master* branches as a measure to avoid
+    potential conflicts between these branches.
+
+    The next version, 0.1.66, is scheduled to be released on February 3rd,
+    with the stabilization phase starting on January 23th, 2022.
+
+    Regards,
+    ```
+
+## During the stabilization
+
+### Bug Fixes
+- Run whatever extra tests you have and report bugs as Upstream issues using the
+[General Issue](https://github.com/ComplianceAsCode/content/issues/new?assignees=&labels=&template=general_issue.md) template.
+    - Propose bug fixes by targeting the PRs to the **stabilization** branch. e.g. `stabilization-v0.1.65`
+    - Once the PR is **merged** in the **stabilization** branch, cherry-pick it to the **master** branch.
+    For example:
+    ```
+    git checkout master
+    git pull upstream master
+    git checkout -b backport_from_stabilization
+    git cherry-pick abcd1234
+    ...
+    ```
+    _NOTE: There are different ways to backport a commit from the stabilization to the master branch._
+    _The above approach is just one example._
+
+In the past, to ensure that all fixes pushed to the stabilization branch were also included in the
+master branch, every Friday during the stabilization phase, a PR was created to merge the
+stabilization changes into the master branch. However, this approach was prone to conflicts that
+could be complex to be solved after one week of asynchronous contributions on master branch.
+Therefore, since the `0.1.63` release, a simpler and more reliable process has been adopted where
+the authors themselves ensure that the fixes are merged into each branch at almost the same time.
+This reduces the risk of conflicts and also makes it easier for authors to resolve them as the
+changes are still "fresh".
+
+### Tests
+
+There is a GitHub Action hooked up with **stabilization** branch that will run a set of tests on every push.
+Make sure that all these tests are passing before moving on with the release process.
 
-Tests during Stabilization Phase
------------
+# Release
 
-There is a GitHub Action hooked up with **stabilization** branch that will run a set of tests on every push.\
-Make sure that all these tests are passing before moving on with the release process.
+## Updating Contributors List
 
-Before the Release
-------------------
+- Update the contributors list by accessing the root folder of Content repository, in the
+**stabilization** branch, and executing the following command:
+    ```
+    PYTHONPATH=. utils/generate_contributors.py`
+    ```
+    - De-duplicate names if necessary.
+    - Make a commit send the a PR.
```

Comment:
The PR to update the contributors list needs to be sent to both master and stabilization branches.

I wonder if it makes sense to update the contributors list before stabilization is branched. :thinking:

Reply:
It would be more efficient to do it before the stabilization branch is created. We can definitely improve this.